unitree_nav/src/cmd_processor.cpp

```diff
@@ -149,6 +155,9 @@ class CmdProcessor : public rclcpp::Node
     high_cmd_.velocity[1] = 0.0;
     high_cmd_.yaw_speed = 0.0;
 
+    // default walking gait is trotting
+    walking_gait = Go1Gait::trot;
```

Comment:
Rename this `walking_gait_` to follow the convention that private class variables have an underscore postfixed to their name.